crates/sui/src/client_commands.rs

```diff
@@ -217,19 +215,6 @@ pub enum SuiClientCommands {
         #[clap(long)]
         signed_tx_bytes: String,
     },
-
-    /// Request gas coin from faucet. By default, it will use the active address and the active network.
-    #[clap[name = "faucet"]]
-    Faucet {
-        /// Address (or its alias)
-        #[clap(long)]
-        #[arg(value_parser)]
-        address: Option<KeyIdentity>,
-        /// The url to the faucet
-        #[clap(long)]
-        url: Option<String>,
-    },
-
```

Comment:
This is the faucet command that we've been advertising, so not sure what went wrong here, but this needs to stay.